pennylane/tape/tape.py

```diff
@@ -82,6 +82,11 @@
 """
 
 
+class TapeError(ValueError):
+    """An error raised when an unsupported operation is attempted with a
+    quantum tape."""
```

Comment:
@antalszava the name of this error doesn't match the docstring 🤔 It sounds like this should be a generic error for something relating to the tape.

Reply:
Updated.